docs/openapi.yml

```diff
@@ -1160,10 +1155,257 @@ paths:
+        type: String
+        enum: ['']
+        description: No user roles needed to access the endpoint
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: service-api-cve-schema.json#/definitions/cnaContainer
+      responses:
+        200:
+          description: Returns the CVE record updated.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    description: Success description
+                  updated:
+                    $ref: service-api-cve-schema.json#/definitions/cnaContainer
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorBadInput'
+        401:
+          description: Not Authenticated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorGeneric'
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorGeneric'
+        404:
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorGeneric'
+        500:
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorGeneric'
+  /cve/{cve-id}/cna/rejected:
+    parameters:
+      - $ref: '#/components/parameters/api-entity-header'
+      - $ref: '#/components/parameters/api-secret-header'
+      - $ref: '#/components/parameters/api-user-header'
+      - $ref: '#/components/parameters/cve-id-path'
+    post:
+      summary: Creates a CVE Record by ID
+      description: |
+        <h2>Access Control</h2>
+        <p>At least one of the following roles are needed to access the endpoint:</p>
+        <p>- <b>SECRETARIAT:</b> The user must belong to an Organization with the “SECRETARIAT” role</p>
+        <h2>Expected Behavior</h2>
+        <p><b>Secretariat:</b> Can create a CVE record owned by any Organization</p>
+      description: TODO
+      operationId: unknown # TODO: add correct operation
+      x-org-roles: # TODO: not sure what goes here
+        type: String
+        enum: ['']
+        description: No Organization roles needed to access the endpoint
+      x-user-roles: # TODO: not sure what goes here
+        type: String
+        enum: ['']
+        description: No user roles needed to access the endpoint
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: service-api-cve-schema.json#/definitions/rejected
```

Comment:
I'm guessing the schema is WIP and we'll add the rejected schema once we actually build it.